tests/std/tests/P0355R7_calendars_and_time_zones_dates/test.cpp

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <chrono>
+#include <type_traits>
+
+using namespace std;
+using namespace std::chrono;
+
+constexpr void day_test() {
+    static_assert(is_trivially_copyable_v<day>, "chrono::day is not trivially copyable");
+    static_assert(is_standard_layout_v<day>, "chrono::day is not standard layout");
+
+    day d1{0u};
+    assert(static_cast<unsigned>(d1) == 0);
+    assert(static_cast<unsigned>(++d1) == 1);
+    assert(static_cast<unsigned>(d1++) == 1);
+    assert(static_cast<unsigned>(d1) == 2);
+
+    assert(static_cast<unsigned>(--d1) == 1);
+    assert(static_cast<unsigned>(d1--) == 1);
+    assert(static_cast<unsigned>(d1) == 0);

```

Comment:
Doing the same thing here for `chrono::month` fails the last assert under constexpr (through `static_assert(test())`) speaks of `__LINE__Var`